web/src/main/java/com/es/phoneshop/web/controller/converter/OrderDtoConverter.java

```diff
@@ -0,0 +1,28 @@
+package com.es.phoneshop.web.controller.converter;
+
+import com.es.core.model.order.Order;
+import com.es.phoneshop.web.controller.dto.OrderDto;
+import org.springframework.core.convert.converter.Converter;
+
+public class OrderDtoConverter implements Converter<OrderDto, Order> {
```

Comment:
Instead of making private order field and using it as simple java class, please make it spring bean.